addons/stock/models/stock_picking.py

```diff
@@ -493,6 +497,11 @@ def _compute_show_validate(self):
             else:
                 picking.show_validate = True
 
+    @api.depends('move_lines.reserved_availability')
+    def _compute_has_reserved_qties(self):
+        for picking in self:
+            picking.has_reserved_qties = any(not float_is_zero(m.reserved_availability, precision_rounding=m.product_uom.rounding) for m in picking.move_lines)
```

Comment:
come on.. if one of the move was partially reserved, the picking goes to partially available